oasislmf/execution/bash.py

```diff
@@ -150,7 +145,14 @@
 }
 trap exit_handler QUIT HUP INT KILL TERM ERR EXIT"""
 
-CHECK_FUNC = """
+def create_check_fucntion(custom_gulcalc_log_start=None, custom_gulcalc_log_finish=None):
```

Comment:
For consistency with other functions that return strings to build up the script, we should name the function `get_...`.

```suggestion
def get_check_fucntion(custom_gulcalc_log_start=None, custom_gulcalc_log_finish=None):
```